File: complexify-android/src/main/java/co/infinum/complexify/ComplexifyBanMode.java
package co.infinum.complexify;

/**
 * Enum for banMode.
 * <p>
 * STRICT - don't allow a substring of a banned password
 * LOOSE - only ban exact matches
 * <p>
 * Created by Damian Marusic on 28.07.15..
 */
public enum ComplexifyBanMode {
    STRICT,
    LOOSE
}

File: complexify/src/main/java/com/infinum/complexify/ComplexifyBanMode.kt
package com.infinum.complexify

/**
 * Which ban mode to use when using the [Complexify.banList].
 */
enum class ComplexifyBanMode {
    /**
     * Don't allow a substring of a banned password.
     */
    STRICT,

    /**
     * Only ban exact matches of a banned password.
     */
    LOOSE
}
